### ISSUE TITLE
Restore Checkpoints error, can't find the key in checkpoint

### ISSUE THREAD

**Issue by rancheng** [dalgu90/resnet-18-tensorflow#5]
OS: Ubuntu 16.04
checkpoint: converted from the torchfile, followed the instruction of the readme.
NotFoundError (see above for traceback): Key conv1/bn/beta/Momentum not found in checkpoint

**Comment by rancheng** [comment]
Fond the resolve, just ignore that Momentum will work:
`variables = tf.global_variables()`
`           vars_restore = [var for var in variables`
`                           if not "logits" in var.name`
`                            and not "Momentum" in var.name]`
`            saver_restore = tf.train.Saver(vars_restore, max_to_keep=10000)`
`            saver_restore.restore(sess, FLAGS.checkpoint)`
